src/pipeline/broad_phase/broad_phase.rs

```diff
@@ -24,6 +24,34 @@ impl ProxyHandle {
     }
 }
 
+/// Proximity handling for BroadPhase updates.
+pub trait BroadPhaseProximityHandler<T> {
```

Comment:
I think we should not use the `Proximity` word as it may end up being confusing since the concept of `Proximity` already exists on ncollide (for configurations where two solids are closer than a given distance). We can name this `BroadPhaseInterferenceHandler` instead.